tests/common.py

```diff
@@ -596,3 +598,62 @@ def ligand_from_smiles(smiles: str, seed: int = 2024) -> Chem.Mol:
     AllChem.EmbedMolecule(mol, randomSeed=seed)
     set_mol_name(mol, smiles)
     return mol
+
+
+def get_guest_params(mol_a, mol_b, atom_map_mixin, q_handle, lj_handle, lamb: float, cutoff: float) -> jax.Array:
```

Comment:
This is not a new function, it is moved from the SingleTopology class but now serves as a reference path for testing purposes.

Reply:
Rather than duplicate here, could this be defined in the `single_topology` module and serve as the implementation of `SingleTopology._get_guest_params`?

Reply:
This function isn't used anymore by anything inside of the SingleTopology class anymore.

Reply:
`SingleTopology._get_guest_params()` has been fully removed.